en/docs/learn/configuring-totp.md

```diff
@@ -0,0 +1,398 @@
+# Configuring TOTP Authenticator
+
+The TOTP authenticator allows you to authenticate a user using
+Time-Based One Time Password (TOTP) through WSO2 Identity Server. It
+uses the TOTP specification to calculate the access tokens based on the
+time and the shared secret key between the user and the identity
+provider.
+
+TOTP is a temporary passcode, generated by an algorithm, for use in 
+authenticating access to computer systems. The algorithm that generates
```

Comment:
How about "TOTP is an algorithm-generated temporary passcode that is used for authentication." ?